test/core/event_engine/test_suite/timer_test.cc

```diff
@@ -38,6 +38,16 @@ class EventEngineTimerTest : public EventEngineTest {
                        std::atomic<int>* fail_count, int total_expected);
 
  protected:
+  void WaitForSignalled(absl::Duration timeout)
+      ABSL_EXCLUSIVE_LOCKS_REQUIRED(mu_) {
+    absl::Time deadline = absl::Now() + timeout;
+    while (!signaled_) {
+      timeout = deadline - absl::Now();
+      GPR_ASSERT(timeout > absl::ZeroDuration());
```

Comment:
Should this be `ASSERT_GT(timeout, absl::ZeroDuration());`?

Reply:
done